src/coreclr/vm/debugdebugger.cpp

```diff
@@ -230,43 +226,221 @@ FCIMPL0(FC_BOOL_RET, DebugDebugger::IsLogging)
 }
 FCIMPLEND
 
-FCIMPL4(void, DebugStackTrace::GetStackFramesInternal,
-        StackFrameHelper* pStackFrameHelperUNSAFE,
-        INT32 iSkip,
-        FC_BOOL_ARG fNeedFileInfo,
-        Object* pExceptionUNSAFE
-       )
+static StackWalkAction GetStackFramesCallback(CrawlFrame* pCf, VOID* data)
```

Comment:
Converted from member function to private in translation unit. This was moved from below.